recipe-box-backend/src/main/java/com/endava/recipebox/service/RecipeService.java

```diff
@@ -0,0 +1,47 @@
+package com.endava.recipebox.service;
+
+
+import com.endava.recipebox.model.Recipe;
+import com.endava.recipebox.model.RecipeStatus;
+import com.endava.recipebox.repository.RecipeRepository;
+import com.endava.recipebox.response.CustomRecipe;
+import com.endava.recipebox.response.CustomRecipeMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class RecipeService {
```

Comment:
Recipe Service should be an interface. And you need to create a class RecipeServiceImpl that implements the interface. And from the controller you will call the interface. The reason to user interfaces is to use one of the OOP concepts: abstraction and decoupling.